Docs/pinout.md

```diff
@@ -30,8 +30,17 @@ The PMOD pins can also be accessed from
 
 ## SPI Pinouts 
 
-For debugging the FPGA <-> SPI <-> Flash/RAM interface it is very helpful to use the 4 through holes on the upper right of the FPGA, and to connect a logic analyzer to the upper right PMOD.  These two diagrams may help.  Remember that the default boot process leaves the Flash switched ``off``.  With the Yosys icepack command, the ``-s`` option boots the Flash in the ``on`` state. 
 
-![SPI Debugging](pinout/debug_spi_black.png)
-![SPI Debugging](pinout/debug_spi_white.png)
```

Comment:
Would it be possible to delete the old files now that the name changed?

Reply:
Great idea. 
Sorry about that. 
Done.

Reply:
No need to be sorry, I am grateful :)